orchestration/dags/dependencies/data_analytics/enriched_data/venue_locations.py

```diff
@@ -12,12 +12,14 @@ def define_table_venue_locations(dataset, table_prefix=""):
             commune_qp as qpv_communes,
             epci.epci_code,
             epci.epci_name,
+            ZRR.CODGEO,
             ZRR.LIBGEO,
             ZRR.ZRR_SIMP,
             ZRR.ZONAGE_ZRR,
-        FROM `{dataset}.QPV`, `{dataset}.{table_prefix}venue` venue, `{dataset}.epci` epci, `{dataset}.ZRR` ZRR
+        FROM `{dataset}.QPV`, `{dataset}.{table_prefix}venue` venue, `{dataset}.epci` epci, `{dataset}.ZRR` ZRR,`{dataset}.correspondance_code_insee` table_insee
         where ST_CONTAINS(geoshape, ST_GEOGPOINT(venue.venue_longitude,venue.venue_latitude)) is true 
               and venue.venue_latitude>-90 and venue.venue_latitude <90 and venue.venue_longitude >-90 
               and venue.venue_longitude <90 and ST_CONTAINS(epci.geo_shape, ST_GEOGPOINT(venue.venue_longitude,venue.venue_latitude)) is true
-              and ZRR.Code_Postal = venue.venue_postal_code;
+              and venue.venue_longitude <90 and ST_CONTAINS(table_insee.geo_shape_insee, ST_GEOGPOINT(venue.venue_longitude,venue.venue_latitude)) is true
```

Comment:
Il y a deux fois le and venue.venue_longitude <90 !

```suggestion
              and ST_CONTAINS(table_insee.geo_shape_insee, ST_GEOGPOINT(venue.venue_longitude,venue.venue_latitude)) is true
```